rfcs/0000-local-overlay-store.md

```diff
@@ -0,0 +1,408 @@
+---
+feature: local-overlay-store
+start-date: 2023-06-14
+author: John Ericson (@Ericson2314)
+co-authors: Ben Radford (@benradf)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
```

Comment:
Thanks to the nominees for accepting! Here's the team the Steering Committee has decided on :)

```suggestion
shepherd-team: @RaitoBezarius @tomberek @arianvp 
shepherd-leader: @tomberek
```